### ISSUE TITLE
GPT Actions getCurrentWeather test fails 

### ISSUE THREAD

**Issue by nkzarrabi** [pplonski/example-weather-app#1]
[debug] Calling HTTP endpoint
[debug] Calling HTTP endpoint
[debug] Response received
{}
Error talking to
It seems there was an error with the request to retrieve the current weather for Boulder, CO. How about we try checking the weather for another location or perhaps I can assist you with planning your climbing trip in another way?

**Comment by pplonski** [comment]
Hi @nkzarrabi,

I've closed the server, you need to run your own server to have access to api endpoint.

**Comment by nkzarrabi** [comment]
I understand but before closing, you might want to change your README
"
What I did? I bought example-weather-app.com domain and set simple FastAPI app with hard-coded JSON response. It is available at https://api.example-weather-app.com/location?location=your+location."
If you close the issue and someone reads and interprets the same way as me, it doesnt do you any good if the repository is public.

**Comment by pplonski** [comment]
You are right, @nkzarrabi. I updated readme. Thank you! May I ask what are you building?

**Comment by nkzarrabi** [comment]
Sure! 
I previously had a GPT in the GPT store but decided to come back to it recently and add a weather API and OpenBeta API ( for climbing database access). 
The purpose of this GPT, named Climbcast GPT, is to provide tailored advice on planning climbing trips based on proximity to your location and the current weather conditions. 

I looked up weather APIs to add to the GPT actions, to offer information on the best times to visit climbing areas but ran into some issues in formatting. Eventually settled on this: 
https://github.com/open-meteo/open-meteo/issues/742#issuecomment-2063299006

Seemed to work for my purposes, where I might ask for weather for a climbing area name (e.g, The Gunks) and ChatGPT has to interpret the data for the API call, ensuring that the query makes sense to the API. (As in the case of "The Gunks", New Paltz in NY state would be the appropriate place to search for weather). 

For the climbing database, I had to adapt a GraphQL API https://api.openbeta.io/
to OpenAPI Schema. 
https://github.com/nkzarrabi/GraphQL-API-to-GPT-Schema

Would be nice to see ChatGPT make the API process a little more refined. 
Very cool that you're affiliated with ChatGPT from this project, they still show your example in the GPT actions. 
Are you working there?
